helm/README.md

```diff
@@ -30,7 +30,6 @@ helm repo add odigos https://odigos-io.github.io/odigos
 ```sh
 helm repo update
 helm upgrade --install odigos odigos/odigos --namespace odigos-system --create-namespace
-kubectl label namespace odigos-system odigos.io/system-object="true"
```

Comment:
need to also update [the docs](https://github.com/odigos-io/odigos/blob/main/docs/setup/installation.mdx#install-odigos-1)

and the [Makefile](https://github.com/odigos-io/odigos/blob/15a2d6fae279b1967f30dee081db76446e801ad4/Makefile#L347)